### PR TITLE
Added charge time to bow shot

### PR DESCRIPTION
**Pull request by kdreese**

### CODE REVIEW

File: src/states/game.gd [hunk 1]
```diff
@@ -228,30 +235,37 @@ func move_to_spawn_point(my_player: CharacterBody3D) -> void:
 	#my_player.get_node("Camera3D").reset_physics_interpolation()
 
 
-func i_would_like_to_shoot(id: String) -> void:
+func setDrawbackIndicator():
```
Comment (admoore0) [inline src/states/game.gd]: 😡

File: src/objects/player.gd [hunk 2]
```diff
@@ -57,7 +63,11 @@ func _unhandled_input(event: InputEvent) -> void:
 		handle_mouse_movement(event as InputEventMouseMotion)
 		get_viewport().set_input_as_handled()
 	elif event.is_action_pressed("shoot"):
-		emit_signal("shoot")
+		drawback_timer = true
+		find_parent("Game").setDrawbackIndicator()
```
Comment (thearst3rd) [inline src/objects/player.gd]: set_drawback_indicator

**Review by thearst3rd — CHANGES_REQUESTED** [review]

File: src/states/game.gd (hunk 1 above)
Comment (thearst3rd) [reply src/states/game.gd]: how dare you copy me in the past

Comment (admoore0) [reply src/states/game.gd]: skill issue

**Review by thearst3rd — APPROVED** [review]

**Review by admoore0 — APPROVED** [review]

**Pull request merged**